novawallet/Common/Helpers/MetaAccountModel+Delegated.swift

```diff
@@ -54,15 +54,25 @@ extension MetaAccountModel {
     func delegationId() -> MetaAccountDelegationId? {
         switch type {
         case .multisig:
-            guard
-                let (chainAccount, multisigModel) = multisigAccount()?.multisig,
-                let multisigModel
-            else { return nil }
+            var multisigModel: DelegatedAccount.MultisigAccountModel?
+            var chainId: ChainModel.Id?
+
+            switch multisigAccount() {
+            case let .universal(multisig):
+                multisigModel = multisig
+            case let .singleChain(chainAccount):
+                multisigModel = chainAccount.multisig
+                chainId = chainAccount.chainId
+            default:
+                return nil
+            }
+
+            guard let multisigModel else { return nil }
 
             return MetaAccountDelegationId(
                 delegatedAccountId: multisigModel.accountId,
                 delegatorId: multisigModel.signatory,
```

Comment:
Shouldn't we use here `delegate` notion. Because as we discussed delegatedAccountId and delegatorId are same